tests/web/web_test.bats

```diff
@@ -0,0 +1,70 @@
+# PURPOSE:
+# I will test the following commands: web create, get, list, update, delete, and clean.
+load ../test_setup.bash
+
+# constants
+FILES=$BATS_TEST_DIRNAME/test_files/*
+
+@test "web create" { # web create test
+  for f in $FILES
+  do
+    run $NIM web create "$f"
+    assert_success
+    assert_output --partial "done"
+  done
+}
+
+@test "web get" { # web get test
+  for f in $FILES
+  do
+    filename=$(basename $f)
+    run $NIM web get -p $filename
+    assert_success
+    assert_output --partial "done"
+    assert_output --partial "getting $filename"
+    assert_output --partial "$(cat $f)" # checking if file content is correct
+  done
+}
+
+@test "web list" { # web list test, 404.html is default in namespaces
+  listString1="404.html"$'\n'"test1.html"
+  listString2="test1.html"$'\n'"404.html"
+
+  run $NIM web list
+  assert_success
+  if [ "$listString1" == "$output" ]; then
+    assert true
+  elif [ "$listString2" == "$output" ]; then
+    assert true
+  else
+   assert false
+  fi
+}
+
+@test "web update" { # web update test
+  for f in $FILES
```

Comment:
It would be good to verify the update command does actually update the file contents accessible via the web url. What about having a second directory of test_files ("updated_files") with the same filenames but different contents. Use the second batch of files with the update and then check the contents is the updated version?

Reply:
Please check this per our conversation on slack